source/index.html

```diff
@@ -23,6 +23,17 @@
     </ul>
 
     <button id="new_comment_button">New Comment</button>
+    <br>
+    <br>
+    <div class="form">
```

Comment:
I would have liked to see the form be added using jQuery rather than editting the HTML